draft-ietf-cdni-ci-triggers-rfc8007bis.xml

```diff
@@ -265,10 +263,10 @@
         <t>
         Once a trigger is created, uCDN can retrieve its trigger state from dCDN or request its state to be modified by 
         transferring an updated representation of the trigger state to dCDN.
-        The CI/T interface supports representation of trigger resources using JSON <xref target="RFC8259"/>.
+        The CI/T interface supports the representation of trigger resources using JSON <xref target="RFC8259"/>.
         </t>
         <t>
-        This RESTful data model built around a common "trigger" resource replaces the command-oriented model 
+        This RESTful data model is built around a common "trigger" resource replaces the command-oriented model 
```

Comment:
This change is grammatically incorrect.  Either the change needs to be reverted, "is" needs to be changed to "that is", or the sentence needs to be split.